content/blog/2019-10-21-rmangal.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: rmangal
+title: 'rmangal: making ecological networks easily accessible'
+date: 2019-10-21
+authors:
+  - Kevin Cazelles
+  - Steve Vissault
+categories: blog
+topicid: 1848
+tags:
+  - Software Peer Review
+  - R
+  - packages
+  - open-data
+  - data
+  - data-access
+  - ecology
+  - ecological networks
+  - API
+  - API client
+  - Mangal
+  - rmangal
+---
+
+
+In early September, the version 2.0.0 of [rmangal][rmangal] was approved by
+rOpenSci, four weeks latter it made the CRAN. Following-up on our experience we
+detail below the reasons why we wrote rmangal, why we submitted our package to
+rOpenSci and how the peer-review improve our package.
+
+
+
+### [Mangal](https://mangal.io/#/), a database for ecological networks
+
+Ecological networks are defined as a set of species populations (the nodes of
+the network) connected through ecological interactions (the edges). Interactions
+are ecological processes in which one species affects another. Although
+predation is probably the most known and documented interaction, other less
+noticeable associations are just as essential to ecosystem functioning. For
+instance, a mammal that unintentionally disperses viable seeds attached to his
+fur might help plants to thrive. All of these interactions occur simultaneously,
+shaping ecosystems functioning and making them as complex as they are
```

Comment:
in "ecosystems functioning" in two places, should there be an apostrophe? "ecosystems' functioning"

Reply:
**ecosystem functioning** is better I think.

Reply:
Changes made in bc4ec8312afa16c90730371cdf250d6bb7203884